src/main/java/com/answerdigital/benhession/academy/answerkingweek2/model/Item.java

```diff
@@ -24,6 +29,11 @@
 @Entity
 @Table(name = "item")
 @JsonIgnoreProperties({"hibernateLazyInitializer"})
+@NoArgsConstructor
```

Comment:
A small detail, but could we move the annotations so that @NoArgsConstructor and @AllArgsConstructor are together please?

Reply:
sure thing, moved it together